src/java/com/rapleaf/jack/queries/WhereConstraint.java

```diff
@@ -54,6 +54,6 @@ public Logic getLogic() {
 
   @Override
   public String getSqlStatement() {
-    return (logic != null ? logic.toString() + " " : "") + "`" + column.getSqlKeyword() + "` " + operator.getSqlStatement();
```

Comment:
Ansi standard uses double quotes for delimited identifiers, which is also supported by postgresql and redshift. However, to use double quotes in mysql, we need to specify the ansi mode: [reference](http://dev.mysql.com/doc/refman/5.0/en/identifiers.html). So it seems that we have to make the query aware of the database type.